code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -316,6 +316,7 @@
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
 		var/obj/item/organ/heart/gland/G = C.getorganslot("heart")
+		var/obj/O = C.get_item_by_slot(slot_head)
```

Comment:
The O var is unecessary

Reply:
Is there a way to do this while still being able to reference the hat item for the messages?

Reply:
if(istype(C.get_item_by_slot(slot_head), /obj/item/clothing/head/foilhat))

Reply:
ohhh, i forgot the istype when I tried that initially. thanks